pkg/apis/kops/dockerconfig.go

```diff
@@ -21,6 +21,7 @@ type DockerConfig struct {
 	LogLevel *string `json:"logLevel,omitempty" flag:"log-level"`
 	IPTables *bool   `json:"ipTables,omitempty" flag:"iptables"`
 	IPMasq   *bool   `json:"ipMasq,omitempty" flag:"ip-masq"`
+	LogOpts  *string `json:"logOpts,omitempty" flag:"log-opt"`
```

Comment:
Can we make `LogOpt` to match the way the other variables are plural vs singular?